admin_ui/src/components/ReferencingTables.vue

```diff
@@ -55,28 +54,32 @@ export default Vue.extend({
                 "fetchTableReferences",
                 this.tableName
             )
-            this.references = (response.data as TableReferencesAPIResponse).references
+            this.references = (
+                response.data as TableReferencesAPIResponse
+            ).references
         },
         clickedReference(reference: TableReference) {
             let columnName = reference.columnName
-            let query = {}
+            let query: any = {}
             query[columnName] = this.rowID
 
-            let location: Location = {
+            let location = {
                 name: "rowListing",
                 params: { tableName: reference.tableName },
-                query
+                query,
             }
+            // updated store to get correct reference table search results
+            this.$store.commit("updateFilterParams", query)
             let vueUrl = this.$router.resolve(location).href
             window.open(
                 `${document.location.origin}${document.location.pathname}${vueUrl}`,
-                "_blank"
+                "_self"
```

Comment:
We might be worth keeping this as "_blank" so it opens in a new tab. Otherwise if they have unsaved form data, it might get lost.

Reply:
I agree with you that `_blank` is better solution, but search result i that case is wrong. Only with opening in same window with `_self` I get correct reference table search results adding [this](https://github.com/sinisaos/piccolo_admin/blob/upgrade_to_Vue3/admin_ui/src/components/ReferencingTables.vue#L71)

Reply:
Ah, I see. Better to keep what you've done then.

Reply:
I managed to use `_blank` and open in new window because is better solution, but I must remove edit option from `RowFormSelect` due to error of `rowID` is `undefined`.